src/pages/Chat/state.tsx

```diff
@@ -28,9 +28,17 @@ export enum Insurer {
   ICA = 'ICA',
   GJENSIDIGE = 'Gjensidige',
   VARDIA = 'Vardia',
+  TRE_KRONOR = 'Tre Kronor',
   OTHER = 'OTHER',
 }
 
+export enum SwitchableInsurers {
```

Comment:
Can't we make this a `ReadonlyArray<Insurer>` instead? It's so easy to make a typo or similar with this code duplication

Reply:
After some irl discussions lets keep it as is, this seems good enough 👍